debug.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import numpy as np
+import time
+from robot import Robot
+
+
+# User options (change me)
+# --------------- Setup options ---------------
+tcp_host_ip = '100.127.7.223' # IP and port to robot arm as TCP client (UR5)
+tcp_port = 30002
+workspace_limits = np.asarray([[0.3, 0.748], [-0.224, 0.224], [-0.255, -0.1]]) # Cols: min max, Rows: x y z (define workspace limits in robot coordinates)
+# ---------------------------------------------
+
+# Initialize robot and move to home pose
+robot = Robot(False, None, None, workspace_limits,
+              tcp_host_ip, tcp_port, None, None,
+              False, None, None)
+
+# Repeatedly grasp at middle of workspace
+grasp_position = np.sum(workspace_limits, axis=1)/2
+grasp_position[2] = -0.25
```

Comment:
Lines 22-25, where did they get these numbers from/what are these numbers for with respect to what?